### PR TITLE
8293214: Add support for Linux/LoongArch64

### PR DESCRIPTION
**Pull request by theaoqi**
Clean backport. Verified on Linux/LoongArch64, Linux/x64 and Linux/aarch64.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8293214](https://bugs.openjdk.org/browse/JDK-8293214): Add support for Linux/LoongArch64

### Reviewers
 * [Johan Vos](https://openjdk.org/census#jvos) (@johanvos - **Reviewer**)
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx17u pull/102/head:pull/102` \
`$ git checkout pull/102`

Update a local copy of the PR: \
`$ git checkout pull/102` \
`$ git pull https://git.openjdk.org/jfx17u pull/102/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 102`

View PR using the GUI difftool: \
`$ git pr show -t 102`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx17u/pull/102.diff">https://git.openjdk.org/jfx17u/pull/102.diff</a>

</details>

### CODE REVIEW

**Comment by kevinrushforth** [comment]
Backports to `jfx17u` require prior approval. You will need to seek such approval from one of the project leads before taking this PR out of Draft.

If it is approved, the timing is such that it needs to wait for about two weeks. It could possibly be slated for JavaFX 17.0.7 (the April CPU release), since it is far too late to make any changes for 17.0.6.

**Comment by theaoqi** [comment]
Thanks a lot for the quick advice, @kevinrushforth. I will further learn how to do the backports to `jfx17u`. Also, this backport is not in a hurry. Let's wait for the release of `17.0.6`. I don't want to bother with the new releasing version:)

**Comment by theaoqi** [comment]
RFC and RFR. Thanks.

**Comment by johanvos** [comment]
@theaoqi Yes, if you take it out of draft, I'll review it.

**Comment by johanvos** [comment]
I'm ok with this.
Apart from the functionality, this BP also makes it easier to backport future changes in build.gradle (because it backports a structural change in the 64-bit "detection")

**Review by johanvos — APPROVED** [review]

**Comment by theaoqi** [comment]
Thanks, @johanvos ! 

Let's wait for a while to see if @kevinrushforth has further comments.

**Pull request closed without merging**
